unity-renderer/Assets/Scripts/MainScripts/DCL/Helpers/Utils/Utils.cs

```diff
@@ -278,6 +278,9 @@ public static GameObject AttachPlaceholderRendererGameObject(UnityEngine.Transfo
 
         public static void SafeDestroy(UnityEngine.Object obj)
         {
+            if (obj is Transform)
```

Comment:
Why we'd need this condition?

Reply:
When the game is being destroyed, for some reason, we are trying to destroy `Transform` components and unity doesn't allow that, throwing an ugly error with no stack trace.  
This condition avoided those error messages.